docs/reference/data-frames/apis/get-transform.asciidoc

```diff
@@ -53,7 +55,11 @@ The API returns the following information:
 `transforms`::
   (array) An array of transform resources.
 
-//==== Authorization
+==== Authorization
+
+If the {es} {security-features} are enabled, you must have
+`data_frame_transforms_user` cluster privileges to use this API. For more
```

Comment:
The specific cluster privilege is `monitor_data_frame_transforms`. A role that has that privilege is `data_frame_transforms_user`